### DISCUSSION TITLE
error NU1100: Unable to resolve 'docx <>=0.0.0>' for 'net7.0/any'

### DISCUSSION THREAD

**Discussion by bbsparkrock** [dotnet/docfx#8990] [General]
When I try to install docfx, I receive the following error;

C:\Users\bburke\AppData\Local\Temp\abc16568-115b-4fc1-b934-c626c9ca0002\restore.csproj : error NU1100: Unable to resolv
e 'docfx (>= 0.0.0)' for 'net7.0'. PackageSourceMapping is enabled, the following source(s) were not considered: Packag
e source.
C:\Users\bburke\AppData\Local\Temp\abc16568-115b-4fc1-b934-c626c9ca0002\restore.csproj : error NU1100: Unable to resolv
e 'docfx (>= 0.0.0)' for 'net7.0/any'. PackageSourceMapping is enabled, the following source(s) were not considered: Pa
ckage source.
The tool package could not be restored.
Tool 'docfx' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-too

**To Reproduce**
1. On a command line, enter dotnet tool install -g docfx

**Expected behavior**
Install to complete successfully.

**Context (please complete the following information):**
- OS: Windows
- Docfx version: Latest version

**Comment by filzrev** [comment]
By following error message.
> error NU1100: Unable to resolve 'docfx (>= 0.0.0)' for 'net7.0/any'. PackageSourceMapping is enabled,
> the following source(s) were not considered: Package source.

It's assumed that `PackageSourceMapping` is customized on your environment,
and failed to resolve `docfx` package by custom package source (name: `Package source`).

What would be the result when running the following command?

> dotnet nuget list source

On my windows environment. following message is displayed.
```
Registered Sources:
  1.  nuget.org [Enabled]
      https://api.nuget.org/v3/index.json
  2.  Microsoft Visual Studio Offline Packages [Enabled]
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
```

I thought, it need to find `NuGet.config` files that affects `PackageSourceMapping`.
and add PackageSourceMapping entry for `docfx` or register `docfx.nupkg` to custom package source. 

**NuGet config file locations**
https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#config-file-locations-and-uses

**Reply by bbsparkrock** [reply]
Thank you! I was able to get version 2.67.5 installed, but when I run docfx commands, it's still using an older version.
PS C:\Program Files\dotnet> dotnet tool update -g docfx
Tool 'docfx' was reinstalled with the latest stable version (version '2.67.5').
PS C:\Program Files\dotnet> docfx --version
docfx 2.59.3.0

How do I get docfx commands to use version 2.67.5? Where is version 2.67.5 installed?

**Reply by filzrev** [reply]
What's the result of following command?

> where docfx
> SET PATH

I thought docfx is installed by other package manager (For example, `chocolatey` or `scoop`).
And `PATH` priority is set higher than .NET Global Tool's path (`%UserProfile%\.dotnet\tools`)

**Comment by bbsparkrock** [comment]
Thank you! The path priority was the issue. I appreciate your help.
